### PR TITLE
feat(supernova): showing scheduled silences in surpressed alerts details and only the longest lasting silence in the alerts list.

### PR DESCRIPTION
**Pull request by TilmanHaupt**
# Summary

Scheduled alerts are now also displayed in the alert details. 
Additionally, the alerts overview now shows only the longest-lasting silence and/or inhibitor.

# Related Issues

fixes https://github.com/cloudoperators/juno/issues/345
closes https://github.com/cloudoperators/juno/issues/384

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: apps/supernova/src/components/alerts/AlertStatus.jsx [hunk 1]
```diff
@@ -18,11 +18,27 @@ const AlertStatus = ({ alert }) => {
...
+  const silence = useMemo(() => {
     if (!alert) return []
-    return getMappingSilences(alert)
+    // sort by biggest endsAt (most far in the future)
+    const silences = getMappingSilences(alert).sort((a, b) => new Date(b.endsAt) - new Date(a.endsAt))
+    return silences.length > 0 ? silences[0] : null
   }, [alert, allSilences, localSilences])
 
+  // Gives inhibitor which will still last the longest
+  const inhibitor = useMemo(() => {
```
Comment (barsukov) [inline apps/supernova/src/components/alerts/AlertStatus.jsx:30]: Do you really need to wrap this thing and previous one in useMemo?

File: apps/supernova/src/components/alerts/AlertStatus.jsx [hunk 2]
```diff
@@ -18,11 +18,27 @@ const AlertStatus = ({ alert }) => {
...
+    if (!alert) return []
+    if (!alert?.status?.inhibitedBy) return []
+
+    let inhibitedBy = alert.status.inhibitedBy.map((fingerprint) => {
```
Comment (barsukov) [inline apps/supernova/src/components/alerts/AlertStatus.jsx:34]: Could you clarify why you mutate same variables here?

**Review by barsukov — CHANGES_REQUESTED** [review]
Let just talk after a call about review

File: apps/supernova/src/components/alerts/AlertStatus.test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getInhibitor } from "./AlertStatus"
+
+describe("getInhibitor", () => {
+  it("Alert to the fingerprint does not exist and should return null", () => {
```
Comment (barsukov) [inline apps/supernova/src/components/alerts/AlertStatus.test.js:9]: please use the formate correctly here

**Review by barsukov — CHANGES_REQUESTED** [review]
Please run prettier on tests, looks like the not right formatting I do not understand why it did not complain on lint

**Review by barsukov — APPROVED** [review]
LGTM!

**Review by andypf — APPROVED** [review]

**Pull request merged**